packages/constants/index.ts

```diff
@@ -0,0 +1,8 @@
+// Partially imported constants from extension repo
```

Comment:
Maybe we need a shared configuration package for things like this? `@leather/config`